yt/frontends/flash/data_structures.py

```diff
@@ -459,7 +459,7 @@ def _is_valid(self, *args, **kwargs):
             fileh = HDF5FileHandler(args[0])
             if "bounding box" in fileh["/"].keys():
                 return True
-        except (IOError, OSError, ImportError):
+        except (OSError, ImportError):
```

Comment:
I genuinely did not know this was redundant.

Reply:
It wasn't always ! Only since python 3.3